cmd/app/link_test.go

```diff
@@ -569,10 +570,8 @@ func Test_Apps_LinkAppHeaderSection(t *testing.T) {
 	}
 	for name, tt := range tests {
 		t.Run(name, func(t *testing.T) {
-			// Setup parameters for test
-			ctx := context.Background()
-
 			// Create mocks
+			ctx := slackcontext.MockContext(t.Context())
 			clientsMock := shared.NewClientsMock()
```

Comment:
note: This is the standard, repetitive change that you'll see. We are swapping `context.Background()` for `slackcontext.MockContext(t.Context)`. 👯